timezone_converter/list_view.py

```diff
@@ -9,12 +9,20 @@
 
 
 class ListView(Helper):
+    def __init__(self, letters: List[str]) -> None:
+        self.letters = letters
+
     def _sort_and_group(self) -> DefaultDict[str, List[str]]:
         sorted_timezones = dict(sorted(self.timezone_translations.items()))
         longest_name = len(max(sorted_timezones, key=lambda x: len(x)))
         timezone_groups: DefaultDict[str, List[str]] = defaultdict(list)
-        for tz_name in sorted_timezones:
-            timezone_groups[tz_name[0]].append(tz_name.center(longest_name))
+        if self.letters == ['all']:
```

Comment:
Here we need to change the comparison in order to accomodate the `const` value change, so we will compare `self.letters` with `[None]`.

Reply:
If we say here ‘return list(string.ascii_letters)’ we could handle the case where a user supplies filtering letter or not exactly the same later. We would just filter by the letters supplied.

Reply:
Sorry, i'm not fully understanding what you mean. Can you elaborate a bit or maybe show a code example or pseudo-code?

Reply:
Let’s say, if the user does not provide letters, to filter by, we could say, we wants to filter by nothing, so all letters a-z are valid. So self.letters would be here [“a”, …, “z”] (which is list(string.ascii_letters)). 
Now we all of a sudden only need the else case of this if statement.

Reply:
Oh, I get what you mean now. Yeah, that's definetely the better approach.

The keys in `sorted_timezones` are always lowercase, and for deduplication purposes, in our case we should use `ascii_lowercase`.

Thanks for the suggestion!